canonical/lxd-ui
# Add volume list and iso imports

## Done

- Add storage volume list
- Add import local file to custom volume
- Add list of images (cached and ISO)

## QA Steps
* launch a new instance and upload a prepared ISO (this needs LXD 5.16 or later, upgrade with `snap refresh lxd --channel=latest/candidate`)
* use the iso image for a new instance
* ensure the instance is launched with a special message (linking to the console tab)
* ensure instance can't be started on creation directly
* check the storage detail page
* check the storage detail page volume tab
* ensure ISO can be deleted on the volume tab if it is not in use.
* check list of images
* try deleting / using a cached image and an iso image
